examples/main.go

```diff
@@ -35,7 +44,18 @@ func main() {
 			return
 		}
 		// Log raw message struct
-		fmt.Println(mes)
+		//fmt.Println(mes)
```

Comment:
yeah, good point. I think thats the better way to do it. Do you think messages should just be thrown away if not caught by a trigger?